### ISSUE TITLE
npm run build not working

### ISSUE THREAD

**Issue by Shizen-no-ko** [moviemasher/moviemasher.js#35]
Hello,

I'm getting the following error when I run npm run build. Could you please tell me what I am doing wrong?

Syntaxfehler.
ERROR: "clean-create" exited with 1.
ERROR: "clean" exited with 1.
npm ERR! Lifecycle script `build` failed with error: 
npm ERR! Error: command failed
npm ERR!   in workspace: example-express-react@5.0.6
npm ERR!   at location: C:\...\moviemasher.js\workspaces\example-express-react 
ERROR: "build-example" exited with 1.

Many thanks in advance for your help.

**Comment by syntropo** [comment]
Hmm... I'm not sure - maybe a file permissions error? The clean-create script is just trying to create the 'dist' directory.

**Comment by syntropo** [comment]
More specifically, it's trying to create a directory several tiers below the 'dist' directory, using `mkdir -p`. You've gotten pretty far into the build process, so I'm guessing it's a problem making those sub directories? Does this not work on windows?

**Comment by Shizen-no-ko** [comment]
Well, it's not working on my windows PC. I have tried to run it a few times but no success so far.

**Comment by syntropo** [comment]
Well, I don't have a windows PC so can't test the problem. Version 5.1 will do all the building within a docker container so may be more useful on the platform.

**Comment by Shizen-no-ko** [comment]
I have managed to get the example-express-react to run. However, image upload works but doesn't display anything on the screen, just stores it locally in public/media, and the upload button works only once. Can you point me to the right place in the docs for filling out this functionality?

**Comment by syntropo** [comment]
I'm afraid there's no documentation yet specific to uploading. You might check out the 5.0.7 branch (which is really 5.1 in the works) since the rendering is working much better there. Hopefully this will be merged in the coming weeks...

**Comment by Shizen-no-ko** [comment]
Hello Doug,
I'm sorry it took so long to get back to you. My client had me doing other things.
I managed to do a workaround to get moviemasher installed on windows.
However, I have now got it running, but when I click the cloud icon and select a video to upload, not much happens after that. Is there something I am missing? Or is it down to me to hook it all up?
I would be grateful for any pointers.
Best regards,
Michael
    Am Freitag, 1. Juli 2022 um 11:47:07 GMT+1 hat Doug Anarino ***@***.***> Folgendes geschrieben:  
 
 


I'm afraid there's no documentation yet specific to uploading. You might check out the 5.0.7 branch (which is really 5.1 in the works) since the rendering is working much better there. Hopefully this will be merged in the coming weeks...
